zcash_client_sqlite/src/wallet.rs

```diff
@@ -716,12 +716,15 @@ pub(crate) fn get_wallet_summary(
         let mut stmt_transparent_balances = conn.prepare(
             "SELECT u.received_by_account, SUM(u.value_zat)
              FROM utxos u
+             LEFT OUTER JOIN transactions tx ON tx.id_tx = u.spent_in_tx
              WHERE u.height <= :max_height
-             AND u.spent_in_tx IS NULL
+             AND (u.spent_in_tx IS NULL OR (tx.block IS NULL AND tx.expiry_height < :chain_tip_height))
```

Comment:
Off-by-one: if the expiry height is equal to the the current tip height and the tx is not yet mined, it has expired.
```suggestion
             AND (u.spent_in_tx IS NULL OR (tx.block IS NULL AND tx.expiry_height <= :chain_tip_height))
```
https://zips.z.cash/zip-0203#specification